pbrd/pbr_map.c

```diff
@@ -721,12 +721,23 @@ void pbr_map_policy_delete(struct pbr_map *pbrm, struct pbr_map_interface *pmi)
 {
 	struct listnode *node;
 	struct pbr_map_sequence *pbrms;
+	bool sent = false;
 
 
 	for (ALL_LIST_ELEMENTS_RO(pbrm->seqnumbers, node, pbrms))
-		pbr_send_pbr_map(pbrms, pmi, false, false);
+		if (!pbr_send_pbr_map(pbrms, pmi, false, true))
```

Comment:
I don't like the `if (!...)` really means it was sent.  Shouldn't it be `if ()` instead?

Reply:
sure